documentation/src/docs/getting-started.md

```diff
@@ -57,7 +57,7 @@ Lets download and run the "Hello World" Pravega sample reader and writer apps. P
 dependencies will be pulled from maven central.
 
 Note: The samples can also use a locally compiled version of Pravega. For more information
-about this see the note on maven publishing in the [README.md](../../../README.md).
+about this see the note on maven publishing in the [README.md](https://github.com/pravega/pravega/blob/master/README.md).
```

Comment:
If this is about samples, then why is it pointing to `pravega/pravega`? It should be pointing to `pravega/pravega-samples`.

Reply:
fixed. it points to `pravega/pravega-samples` now.